app/__init__.py

```diff
@@ -30,5 +30,9 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .routes import tasks_bp
+    from .routes import goals_bp
+    app.register_blueprint(tasks_bp)
+    app.register_blueprint(goals_bp)
```

Comment:
You can combine imports to one line like so:
```suggestion
    from .routes import tasks_bp, goals_bp

    app.register_blueprint(tasks_bp)
    app.register_blueprint(goals_bp)
```